### DISCUSSION TITLE
"Audio Gridder failed to bind to the server port"

### DISCUSSION THREAD

**Discussion by spittinpigeon** [apohl79/audiogridder#617] [Help] [answered]
Hello all.
Thanks for any help that may be provided. I'm having trouble getting past this step. Spent about 10 hours trying everything I could think of. Allowed the program through the firewall, as well as the port #. Tried many youtube tips and tricks no no avail.
It worked ONCE, and even then, I couldn't load a plugin because it said improper routing.
I'm just about at the end of the allotted time I can give up in my life for this project. HELP!!! lol
Thank You!!!

![Screenshot (348)](https://user-images.githubusercontent.com/86077280/122457458-abca7a80-cf7c-11eb-9e85-11b8b8f6dee7.png)

**Comment by CaBuffalo** [comment, marked as answer]
There is an instance running in the back ground. So you can't load AG again.

Take a look at your task manager (Windows OS) or activity monitor (MACOS).
Kill the AG instance and you will be able to restart it.

On Thu, Jun 17, 2021 at 12:05 PM spittinpigeon ***@***.***>
wrote:

> Hello all.
> Thanks for any help that may be provided. I'm having trouble getting past
> this step. Spent about 10 hours trying everything I could think of. Allowed
> the program through the firewall, as well as the port #. Tried many youtube
> tips and tricks no no avail.
> It worked ONCE, and even then, I couldn't load a plugin because it said
> improper routing.
> I'm just about at the end of the allotted time I can give up in my life
> for this project. HELP!!! lol
> Thank You!!!
>
> [image: Screenshot (348)]
> <https://user-images.githubusercontent.com/86077280/122457458-abca7a80-cf7c-11eb-9e85-11b8b8f6dee7.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/617>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FWYVGWUGZKWTNWSMDTTJBO5ANCNFSM464IGCAA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by spittinpigeon** [reply]
Thank You!

**Comment by Macaroni108** [comment]
That happened to me last night as well and it's when using Sandboxing and one or more of the individual Sandboxes is hanging up everything - Force Quit - as was mentioned above.

**Reply by spittinpigeon** [reply]
Thank you!

**Reply by Henri75** [reply]
Got the same issue on windows.
Happens almost every time when I restart the server.
I need to kill all instances in task manager.

**Reply by mcaudio** [reply]
I'm having the same problem, but when I kill the instances AG won't reopen. Any thoughts?

**Comment by NutsoSix** [comment]
When Audio Gridder is trying to do the Scan, go to the windows toolbar and click on the Audio Gridder icon and go to server settings and uncheck everything then you will be able to get it to work.
